docs/src/pages/it/getting-started.md

```diff
@@ -0,0 +1,70 @@
+
+In alternativa, leggi la nostra [Guida all'Installazione](/installation) per una panoramica più completa su come impostare Astro.
+
+### Esempi
+
+Se preferisci imparare Astro tramite esempi, dai un'occhiata alla nostra [libreria completa degli esempi](https://github.com/snowpackjs/astro/tree/main/examples) su GitHub.
+
+Puoi testare uno qualsiasi di questi esempi sulla tua macchina locale lanciando `npm init astro` con la flag `--template` da CLI. La flag `--template` supporta anche template di terze parti della community.
+
+```bash
+# Avvia la procedura guidata e usa questo template ufficiale
+npm init astro -- --template [OFFICIAL_EXAMPLE_NAME]
+# Avvia la procedura guidata e usa questo template della community
+npm init astro -- --template [GITHUB_USER]/[REPO_NAME]
+npm init astro -- --template [GITHUB_USER]/[REPO_NAME]/path/to/example
+```
+
+### Online Playgrounds
+
+Se ti interessa provare Astro in un browser, puoi utilizzare un code editor online come Stackblitz, CodeSandbox, Gitpod o GitHub Codespaces. Clicca sul link "Open in Stackblitz" in uno dei nostri esempi nella [libreria degli esempi](https://github.com/snowpackjs/astro/tree/main/examples). In alternativa, [clicca qui](https://stackblitz.com/fork/astro) per avviare un nuovo progetto in [Stackblitz](https://stackblitz.com/fork/astro).
+
+## Impara Astro
+
+In Astro arrivano persone di tutti i tipi, con il loro background di conoscenze e stili d'apprendimento diversi. Sia che tu preferisca un approccio più teorico o più pratico, speriamo troverai questa sezione utile.
+
+- Se preferisci **imparare facendo cose**, puoi iniziare con la nostra [libreria degli esempi](https://github.com/snowpackjs/astro/tree/main/examples).
+- Se preferisci **imparare i concetti passo-passo**, puoi iniziare da [guide e concetti base](/core-concepts/project-structure).
+
+Come ogni tecnologia poco familiare, Astro si presenta con una leggera curva di apprendimento. Tuttavia, con un po' di pratica e pazienza, sappiamo che ci riuscirai in poco tempo.
+
+## Imparare la Sintassi `.astro`
+
+Quando inizierai ad imparare Astro, vedrai molti file usare l'estensione `.astro`. Questa è la **Sintassi dei Componenti di Astro**: è un file speciale simile all'HTML che Astro usa per i template. È stato progettato per risultare familiare a chiunque abbia avuto esperienze con l'HTML o JSX.
+
+La nostra utile guida ai [componenti Astro](/core-concepts/astro-components) ti aiuterà a comprendere la sintassi Astro, ed è il modo migliore per impararla.
+
+### Reference API
+
+Questa parte del documento è utile quando si vuole comprendere meglio una particolare API di Astro. Ad esempio, la [Reference di Configurazione](/reference/configuration-reference) elenca tutte le possibili opzioni di configurazione possibili. La [Reference dei componenti incorporati](/reference/builtin-components) elenca tutti i componenti principali disponibili, come `<Markdown />` e `<Code />`.
```

Comment:
Here “documento” is used rather than “documentazione”. Wouldn’t “documento” suggest the current page?

```md
Questa sezione della documentazione è utile...
```